examples/classification/requirements.txt

```diff
@@ -0,0 +1,12 @@
+requests==2.25.1
```

Comment:
nit: can delete [requests, tqdm, numpy, onnx, sparsezoo], they come with deepsparse. let's also make deepsparse the first requirement and remove the `0.5.1` requirement for deepsparse and sparseml

Reply:
🌊  done